src/azure-cli/azure/cli/command_modules/backup/custom_wl.py

```diff
@@ -257,6 +257,9 @@ def show_protectable_item(items, name, server_name, protectable_item_type):
 
 
 def show_protectable_instance(items, server_name, protectable_item_type):
+    if server_name is None:
+        raise RequiredArgumentMissingError("Server name missing. Please provide a valid server name.")
```

Comment:
To make the message more actionable, we may also tell users which argument they can use to pass a valid workload type

Reply:
Actually, this function is used in other commands too so the argument can be either --server-name or --target-server-name depending on the command used.

Reply:
Can we split the cases or pass the function an additional parameter to differentiate the arguments?

Reply:
Pushed the relevant changes. Please have a look.